src/v2/cookbook/form-validation.md

```diff
@@ -333,21 +333,34 @@ We set up the total value as a computed value, and outside of that bug I ran int
 
 ## Server-side Validation
 
-In my final example, we built something that makes use of Ajax to validate at the server. The form will ask you to name a new product and will then check to ensure that the name is unique. We wrote a quick [OpenWhisk](http://openwhisk.apache.org/) serverless action to do the validation. While it isn't terribly important, here is the logic:
+In my final example, we built something that makes use of Ajax to validate at the server. The form will ask you to name a new product and will then check to ensure that the name is unique. We wrote a quick [Netlify](https://netlify.com/) serverless action to do the validation. While it isn't terribly important, here is the logic:
 
 ``` js
-function main(args) {
-    return new Promise((resolve, reject) => {
-        // bad product names: vista, empire, mbp
-        const badNames = ['vista', 'empire', 'mbp'];
-
-        if (badNames.includes(args.name)) {
-          reject({error: 'Existing product'});
-        }
+exports.handler = async (event, context) => {
+  
+    const badNames = ['vista', 'empire', 'mbp'];
+    let name = event.queryStringParameters.name;
+
+    if (badNames.includes(name)) {
+      return { 
+        statusCode: 200,         
+        headers:{
```

Comment:
Can you please lint this code snippet? There are several linting errors.

Reply:
Considering this is very much *not* the focal point of the cookbook entry, I don't think is really necessary. In fact, I'd rather folks not even look at the server-side code as the focus is on the Vue aspect.

Reply:
It doesn't need to be the focal point. If we have code in our official cookbook, it better looks good.

Reply:
Sure, but again, this is *not* the point of the article. We don't know what backend folks are using. Heck, if I had my way I'd hide it completely and in fact, the more that I think about it the more I'd like to do that. @sdras - do you have an opinion on this? While I agree with some of the comments about the code here, I do not agree with all, and it feels like a waste of time to get deep into it. Could we just simplify things and *not* show the back end code?

Reply:
I don't think you got what I mean. I do agree that this code snippet isn't the point of the article, and sure, if we can hide it I won't bother. But we're showing it here, and if it's shown, it should be in its best shape.

Reply:
Hmm ok maybe. I'd still like to see what @sdras thinks about just skipping the code completely.

Reply:
Hi @cfjedimaster, please respond to @phanan's comments here. It's not the right path to just delete parts of the examples like this. An spent the time to review, and his suggestions are great. The point of the cookbook is an extension past Vue to use cases, it's not necessarily only Vue code we're aiming for here. 

Can you please go through the PR and work through An's suggestions? Thanks.

Reply:
Ok, I wasn't trying to disregard his comments, but I honestly thought it would simplify things to focus on the client-side code. I'll bring it back (later today anyway).